pkgs/applications/video/mplayer/default.nix

```diff
@@ -100,8 +102,10 @@ stdenv.mkDerivation rec {
     rm -rf ffmpeg
   '';
 
+  depsBuildBuild = [ buildPackages.stdenv.cc ];
```

Comment:
Since when is this an API? No results in `git grep`...

Reply:
That is my mistake. This is an API I intended to add, but hadn't yet. This change was not supposed to make this PR.